actions/get_switches.py

```diff
@@ -35,24 +35,25 @@ def run(self, ipaddress=None, username=None, password=None):
         client = CFMClient(ipaddress, username, password)
 
         # Get switches from plexxi controller
-        bunch_of_switches = fabric.get_switches(client)
+        try:
+            bunch_of_switches = fabric.get_switches(client)
+        except:
+            error = "ERR-LOGIN - Failed to log into CFM controller"
+            return error
+
+        # Setup a list for holding values
         switch_data = []
-        c = 0
+
         # Process switch datat from plexxi API
         for i in bunch_of_switches:
```

Comment:
'i' is usually used for indexing; here you are iterating through a collection of objects. "sw" or "cmf_switch" might be clearer to the reader, but not "switch" as that's a reserved word :-0

Reply:
agreed